app/assets/javascripts/activeadmin_addons/inputs/nested-select.js

```diff
@@ -150,6 +150,9 @@ $(function() {
 
       if (!!parent) {
         var parentSelectorId = '#' + model + '_' + parent;
+				if (!$(parentSelectorId).length) {
+					parentSelectorId = container.find('*[id*='+ parent +']')[0];
```

Comment:
Expected indentation of 2 spaces but found 5 tabs  indent<br>Infix operators must be spaced                     space-infix-ops